PVEntityGenerator/PVEntityGenerator.csproj

```diff
@@ -81,6 +81,29 @@
     <PlatformTarget>x86</PlatformTarget>
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|x86'">
```

Comment:
this new block should not be required for this PR?

Reply:
ok, removed